metaplex/js/test/store.test.ts

```diff
@@ -0,0 +1,36 @@
+import { SetStore } from '../src/transactions';
+import { Store } from '../src/accounts/Store';
+import { FEE_PAYER, NETWORK } from './utils';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { NodeWallet } from './wallet';
+import { airdrop } from '@metaplex-foundation/amman';
+
+describe('Store', () => {
+  let connection: Connection;
+  jest.setTimeout(80000);
+
+  beforeAll(async () => {
+    connection = new Connection(NETWORK);
+    await airdrop(connection, FEE_PAYER.publicKey, 2);
+  });
+
+  describe('Store', () => {
+    test('SetStore', async () => {
+      const wallet = new NodeWallet(FEE_PAYER);
```

Comment:
In a proper test this should probably not be the FEE_PAYER but rather a randomly generated wallet? Also not sure I should've copied NodeWallet as is, just FYI.